src/main/java/ch/securify/OutputGenerator.java

```diff
@@ -0,0 +1,142 @@
+/*
+ *  Copyright 2018 Secure, Reliable, and Intelligent Systems Lab, ETH Zurich
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package ch.securify;
+
+import java.util.*;
+import java.io.*;
+
+
+public class OutputGenerator {
+    private static final String ANSI_RESET = "\u001B[0m";
+    private static final String ANSI_RED = "\u001B[31m";
+    private static final String ANSI_GREEN = "\u001B[32m";
+    private static final String ANSI_YELLOW = "\u001B[33m";
+
+
+
+    /**
+     * Print the contract analysis output using syntax similar to clang style errors.
+     *
+     * @param the result from the contract analysis
```

Comment:
Glad that you wrote javadoc! Unfortunately this doesn't seem to be the exact syntax, you're missing the name of the parameter, maybe you can change the settings of your IDE to have some visual cue about this to fix everything quickly.

Reply:
Yes sorry, I pretty much never write Java so I just quickly hacked together the docs. I will double check the proper style and fix it.